mne/epochs.py

```diff
@@ -3366,7 +3334,7 @@ def _concatenate_epochs(epochs_list, with_data=True, add_offset=True):
     shift = int((10 + tmax) * out.info['sfreq'])
     events_offset = _update_offset(None, out.events, shift)
     for ii, epochs in enumerate(epochs_list[1:], 1):
-        _compare_epochs_infos(epochs.info, info, ii)
+        _ensure_infos_match(epochs.info, info, f'epochs[{ii}]')
```

Comment:
I think you should also add `on_mismatch` to `concatenate_epochs`, since it's very similar to raw. I guess it should also be done for evoked?

Reply:
Ok!